tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
     "extends": "./.svelte-kit/tsconfig.json",
     "compilerOptions": {
-        "lib": ["webworker"],
+        "lib": [
+            "webworker",
+            "DOM",
+            "ESNext"
```

Comment:
Do we even need those? I am pretty sure they are present in `"extends": "./.svelte-kit/tsconfig.json"`

Reply:
They are, but by defining lib as webworker only, we're overriding svelte-kit's tsconfig's lib option, so we need to re-include it.

Reply:
ah okay, thought its actually extending :-D